README.md

```diff
@@ -110,26 +110,26 @@ sudo pkg add nginx-agent-<agent-version>
 ```
 
 ## Starting and Enabling Start on Boot
-To start the Agent on systemd systems, run the following command:
+To start the NGINX Agent on systemd systems, run the following command:
 ```
 sudo systemctl start nginx-agent
 ```
-To enable the Agent to start on boot, run the following command:
+To enable the NGINX Agent to start on boot, run the following command:
 ```
 sudo systemctl enable nginx-agent
 ```
 
 ## Logging 
-Agent uses formatted log files to collect metrics. Expanding log formats and instance counts will also increase the size of Agent log files. We recommend adding a separate partition for `/var/log/nginx-agent`. Without log rotation or storage on a separate partition, log files could use up all the free drive space and cause your system to become unresponsive to certain services.
+NGINX Agent uses formatted log files to collect metrics. Expanding log formats and instance counts will also increase the size of NGINX Agent log files. We recommend adding a separate partition for `/var/log/nginx-agent`. Without log rotation or storage on a separate partition, log files could use up all the free drive space and cause your system to become unresponsive to certain services.
 
-# Getting Started with Agent
-Follow these steps to configure and run Agent and a mock interface ("control plane") to which the Agent will report.
+# Getting Started with NGINX Agent
+Follow these steps to configure and run NGINX Agent and a mock interface ("control plane") to which the NGINX Agent will report.
 
-## Installing NGINX and Agent
-Follow steps in the [Installation](#installation) section to download, install, and run NGINX and Agent.
+## Installing NGINX and NGINX Agent
+Follow steps in the [Installation](#installation) section to download, install, and run NGINX and NGINX Agent.
 
-## Cloning the Agent Repository
-Run the following command in your development directory to clone the Agent source code from the GitHub repository. See [Cloning a GitHub Repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) for additional help.
+## Cloning the NGINX Agent Repository
+Run the following command in your development directory to clone the NGINX Agent source code from the GitHub repository. See [Cloning a GitHub Repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) for additional help.
 
```

Comment:
@mtbChef think the README doesn't need to go into detail for cloning. Each contributor will have their own preference for cloning the repo. Maybe we should have it as:
```
Clone the NGINX Agent source code into your development directory from the GitHub repository using your preferred cloning method.  See [Cloning a GitHub Repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) for additional help.
```

Reply:
@craigell that's a good point. https://github.com/nginx/agent/pull/126/commits/a65821ce6261745ac1c8023cebf5c891be314334 solves this. I've simplified the sentence a bit more for clarity.

Reply:
https://github.com/nginx/agent/pull/126/commits/33a9c24f6cc439d9d54860f26ec3dd085961fb05 removes duplicate cloning instructions under Development Environment Setup and links to above section discussed in thread. Fixes table of contents as well.